### ISSUE TITLE
Search feature improvement: Pin coordinates

### ISSUE THREAD

**Issue by d4f5409d** [openstreetmap/iD#9415]
### Description

I do often edit stuff using coordinates. And when I paste into iD the coordinates I need, iD shows only the area of that coordinate, but it doesn't indicate anything else. I would like to get the feature where it shows where exactly that coordinate is located, similar to OSMAnd, so I could make a POI right there, exactly at that coordinate.

### Screenshots

![IMG_20221210_121642](https://user-images.githubusercontent.com/76624594/206852313-06e1e06b-b158-4d73-b5fd-79b94eceea3d.jpg)
![IMG_20221210_121709](https://user-images.githubusercontent.com/76624594/206852322-990f3ca2-ea1d-4673-a6ab-13cee2e9b46e.jpg)

**Comment by tordans** [comment]
FYI: There are many comments about this in at least one, maybe multiple older issues.

**Comment by tyrasd** [comment]
yeah, this has been discussed already. see for example https://github.com/openstreetmap/iD/issues/11471#issuecomment-3400552806 or #4197
